requirements.txt

```diff
@@ -1,12 +1,12 @@
-certifi==2020.4.5.1
 babel==2.7.0
 flask-babel==1.0.0
 flask==1.1.2
-idna==2.9
 jinja2==2.11.1
 lxml==4.5.0
 pygments==2.1.3
-pyopenssl==19.1.0
 python-dateutil==2.8.0
 pyyaml==5.3.1
-requests[socks]==2.23.0
+requests[socks]==2.24.0
+idna==2.10
+pyopenssl==19.1.0
+certifi==2020.6.20
```

Comment:
If we keep this sorted version updates are easier to spot.

Reply:
The order in this file is basically random, the version above tries to group by theme (flask, http,...).
But anyway this is small file.
I've have pushed force a new commit that keep the same.

Reply:
> The sort order in this file is basically random

Is it? In f9186344 there was an alphabetic order and in a3368c07a babel was added at the wrong position.